src/000-SCRIPT_OBJ/Player.js

```diff
@@ -1158,24 +1163,52 @@ App.Entity.Player = class Player {
     /**
      * Returns the config settings for a statistic type.
      * @param {string} Type - STAT | SKILL | BODY
-     * @returns {*}
+     * @returns {object}
      */
     GetStatConfig (Type) {
         if (Type == "STAT") return App.Data.Lists.StatConfig;
         if (Type == "SKILL") return App.Data.Lists.SkillConfig;
         if (Type == "BODY") return App.Data.Lists.BodyConfig;
-    };
+    }
+
+    /**
+     * Returns object that holds stat values
+     * @param {string} Type
+     * @returns {object}
+     */
+    GetStatObject(Type) {
+        if (Type == "STAT") return this._state.CoreStats;
+        if (Type == "SKILL") return this._state.Skills;
+        if (Type == "BODY") return this._state.BodyStats;
+    }
+
+    /**
+     * Returns object that holds stat XP values
+     * @param {string} Type
+     * @returns {object}
+     */
+    GetStatXPObject(Type) {
+        if (Type == "STAT") return this._state.CoreStatsXP;
+        if (Type == "SKILL") return this._state.SkillsXP;
+        if (Type == "BODY") return this._state.BodyXP;
+    }
+
     /**
      * Return a statistic value (raw)
      * @param {string} Type
      * @param {string} StatName
      * @returns {number}
      */
     GetStat (Type, StatName) {
-        if (Type == "STAT") return this._state.CoreStats[StatName];
-        if (Type == "SKILL") return this._state.Skills[StatName];
-        if (Type == "BODY") return this._state.BodyStats[StatName];
-    };
+        var statObj = this.GetStatObject(Type);
```

Comment:
This change and the functions above were introduced to allow for new stats additions without save data migration.